### DISCUSSION TITLE
"unlisted" visibility is not intuitive

### DISCUSSION THREAD

**Discussion by jeriox** [Traewelling/traewelling#2056] [Ideas / Feature Requests]
I used the "unlisted" visibility for the first time today. I expected a behaviour similar to how unlisted works on YouTube etc.: when viewing a dashboard or the profile, the item is not visible. Everyone who has the link to the item can view it via the link. I think this is a really cool behaviour for trips that I don't want to publish but show to selected friends by sending the link to them. On the other hand, I don't really understand what the current behaviour of unlisted should achieve and I certainly dind't expect it to behave this way judging by the name. I therefor propose to use the unlisted visibility as described above.
Also I have been told that other people are not able to star the unlisted status/mark it as favorite

**Comment by rugk** [comment]
You may be interested in https://github.com/Traewelling/traewelling/discussions/2508
